styleguide/screens/Patterns/Medallion/Medallion.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Medallion from '../../../../components/Medallion/Medallion';
+import Specimen from '../../../components/Specimen/Specimen';
+import { H, T, C } from '../../../components/Scaffold/Scaffold';
+
+import m from '../../../../globals/modifiers.css';
+
+const MedallionDocumentation = () => (
+  <div>
+    <H level={ 1 }>Medallion</H>
+    <T elm="p" className={ cx(m.mtr, m.largeI, m.demi) }>
+      Medallions are used to display a numerical value alongside  other
+      information, such as a label.
+    </T>
+    <T elm="p" className={ m.mtLgIi }>
+      Medallions currently exist in 2 variations, <C>light</C> for dark
+      backgrounds, and <C>dark</C> for lighter backgrounds.
+    </T>
+    <Specimen
+      classNames={ {
+        root: m.mtr,
+        specimenContainer: m.par,
+      } }
+      code={ dedent`
+        <Medallion>10</Medallion>
```

Comment:
@jordanamblin We might want to consider how we show components with white backgrounds inside the specimen component. At the moment they blend in to the point you can't really see it 😕